src/util/str_to_bool.cpp

```diff
@@ -0,0 +1,19 @@
+#include "util/str_to_bool.h"
+#include "util/format.h"
+#include <string>
+
+namespace bpftrace::util {
+
+bool is_str_bool_truthy(const std::string &value)
```

Comment:
We have this functionality in the `ConfigParser<bool>`. I think as a separate change, can you generalize from `ConfigParser<...>` to `Parser<...>` and we can just have a `util/parsers.h` or `parsers/bool.h` with these implementation? Just trying to avoid a return to the old `util` where there's a lot of overlapping/duplicate functionality.

Reply:
See above. ConfigParser call this function now. I can change the name to parsers.h though.